src/components/VTeleport/VTeleportTarget.vue

```diff
@@ -20,11 +21,17 @@ export default defineComponent({
     targets[this.name] = this
   },
   beforeDestroy() {
-    delete targets[this.name]
-    if (this.children.length > 0)
+    if (this.children.length > 0) {
+      if (this.forceDestroy) {
+        this.children.forEach((child) => child.$destroy())
```

Comment:
I've read somewhere that if you call `$destroy()` yourself, you have to also manually delete the HTML element.

Reply:
:open_mouth: Good call!